web/src/main/java/org/cbioportal/web/StructuralVariantController.java

```diff
@@ -66,7 +66,8 @@ public ResponseEntity<List<StructuralVariant>> fetchStructuralVariants(
             @RequestAttribute(required = false, value = "involvedCancerStudies") Collection<String> involvedCancerStudies,
             @Valid @RequestAttribute(required = false, value = "interceptedStructuralVariantFilter") StructuralVariantFilter interceptedStructuralVariantFilter,
             @ApiParam(required = true, value = "List of entrezGeneIds and molecularProfileIds or sampleMolecularIdentifiers")
-            @Valid @RequestBody(required = false) StructuralVariantFilter structuralVariantFilter) {
+            @Valid @RequestBody(required = false) StructuralVariantFilter structuralVariantFilter
```

Comment:
Let's remove this whitespace change from the git diff

Reply:
Fixed in [52b680d](https://github.com/cBioPortal/cbioportal/commit/52b680d6bf158a8f0f4308e4fa385921110aa5b8)